### ISSUE TITLE
Console support (Xbox & PlayStation)

### ISSUE THREAD

**Issue by repi** [EmbarkStudios/rust-ecosystem#18]
We would like to have support for Rust on both Xbox and PlayStation. Platforms we are intimately familiar with as many of our devs have been working on them for the last 15 years but in C++.

NOTE: As the development environments and licensing restrictions on the console is private, we can't share or discuss details openly here. But want to have a public ticket to track our interest at least.

**Comment by repi** [comment]
Also tracked in the Rust Gamedev WG https://github.com/rust-gamedev/wg/issues/90

**Comment by Jake-Shadle** [comment]
https://rust-embedded.github.io/embedonomicon/custom-target.html bookmarking creation of custom targets.

**Comment by rlabrecque** [comment]
We have reached out to our various account managers and expressed some interest to them directly.

**Comment by dlight** [comment]
@repi 

> NOTE: As the development environments and licensing restrictions on the console is private, we can't share or discuss details openly here. But want to have a public ticket to track our interest at least.

Does this means that console support won't be open source?

**Comment by repi** [comment]
@dlight correct, it can't be open source unless Microsoft and/or Sony explicitly allows it to be, which we would plead for but is highly unlikely to happen.

**Comment by DemiMarie** [comment]
> @dlight correct, it can't be open source unless Microsoft and/or Sony explicitly allows it to be, which we would plead for but is highly unlikely to happen.

UWP support can be open source!
